examples/dreambooth/train_dreambooth.py

```diff
@@ -326,16 +360,43 @@ def get_full_repo_name(model_id: str, organization: Optional[str] = None, token:
         return f"{organization}/{model_id}"
 
 
+def save_model(accelerator, unet, text_encoder, args, step=None):
```

Comment:
I have a hard time seeing how this is related to the PR of adding an option for labeled samples? Could we maybe try to keep PRs laser focused on one thing? :-)